server/server.go

```diff
@@ -2,6 +2,7 @@ package server
 
 import (
 	"database/sql"
+	_ "github.com/lib/pq"
```

Comment:
I don't think this should be here - global `_` imports should only exist in `main`. It's [already imported anyway](https://github.com/bobheadxi/calories/blob/dev/main.go#L10) - please switch to Visual Studio Code and use a real Go linter.